model/Security/Classes/VexNotAffectedVulnAssessmentRelationship.md

```diff
@@ -4,25 +4,49 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+Links a vulnerability and one or more elements designating the latter as products
+not affected by the vulnerability.
 
 ## Description
 
-An VexNotVulnAffectedAssessment is TODO
+`VexNotVulnAffectedAssessmentRelationship` connects a vulnerability and a number
+of elements desigating them as products not affected by the vulnerability.
+This relationship corresponds to the VEX `not_affected` status.
+
+## Constraints
+
+When linking elements using a `VexNotVulnAffectedAssessmentRelationship`, the
+following requirements must be observed:
+
+* Relating elements with a `VexNotVulnAffectedAssessmentRelationship` is restricted
+to the `doesNotAffect` relationship type.
+* The `from:` end of the relationship must be a `/Security/Vulnerability` classed
+element.
+* Both `impactStatement` and `justification` properties have a cardinality of
```

Comment:
Should a document not satisfying this constraint here be treated as invalid? Then this should probably be put into a more structured form, so that the generation picks it up.

(random note: this could also be modeled as one or more of type `impactStatement_or_Justification` or something similar)